test-suite/squidconf/tls-certs.conf

```diff
@@ -0,0 +1,257 @@
+## Copyright (C) 1996-2018 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+#
+# The following port configurations check all 50 possible permutations of these
+# configs. Among those, the negative ones (unsupported and invalid) are
+# currently commented out. TODO: Check each negative test case in isolation.
+#
+#  A) certificate(s):
+#    1) leaf only         tls-cert=./leaf-rsa.*
+#    2) full chain        tls-cert=./leaf-root-rsa.*
+#    3) root CA           tls-cert=./ca-root-rsa.*
```

Comment:
For consistency with "leaf only" in item 1.

```suggestion
#    3) root CA only      tls-cert=./ca-root-rsa.*
```

Alternatively, we can drop "only" from all entries.